gpu-operator/troubleshooting.rst

```diff
@@ -649,16 +658,21 @@ EFI Secure Boot is currently not supported with the GPU Operator
 
 Disable EFI Secure Boot on the server.
 
-File an issue
-=================
+***************************************************************************************
+GPU Operator pods in ``Init:RunContainerError`` or ``Init:CreateContainerError`` state
```

Comment:
Question -- do we want to add this to our docs given that we are fixing this bug in 25.10.1?